src/screens/SavedEventListScreen/NoSavedEvents.js

```diff
@@ -52,6 +54,9 @@ const styles = StyleSheet.create({
     flex: 3,
     marginHorizontal: 12
   },
+  pad: {
+    marginBottom: 24 // padding when you scroll to bottom
```

Comment:
Looks like in other views we generally use 12px for the bottom padding.

See:
* https://github.com/redbadger/pride-london-app/blob/5ee29f70bb4169cbab17c593cdf471c107b6a1cd/src/screens/EventDetailsScreen/component.js#L191
* https://github.com/redbadger/pride-london-app/blob/3b20821e87dbd13b1cf2be73bc9e3b4ec5387b14/src/screens/SponsorScreen/component.js#L155
* https://github.com/redbadger/pride-london-app/blob/3b20821e87dbd13b1cf2be73bc9e3b4ec5387b14/src/screens/FilterModal/component.js#L117

Reply:
it just looked a little small to me, but fair enough! Will change